cluster/watchdog.go

```diff
@@ -121,6 +121,9 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 			}
 		}
 
+		// Clear out the network configs, since we cannot create a
+		// container with multiple networks.
+		c.Config.NetworkingConfig.EndpointsConfig = nil
```

Comment:
We weren't disconnecting/reconnecting networks for local networks before. In fact, doing so for the `local` networks is almost guaranteed to fail since we're rescheduling this container on a new node, which will not have the local networks the container previously relied upon.